client/src/components/CommentApp/components/Comment/index.tsx

```diff
@@ -281,11 +274,11 @@ export default class CommentComponent extends React.Component<CommentProps> {
       <>
         <CommentHeader commentReply={comment} store={store} strings={strings} />
         <form onSubmit={onSave}>
-          <textarea
+          <TextArea
```

Comment:
You've doubled up on the focusing logic here - we already focus in `ComponentDidMount`. Bearing in mind we're going to need to move keyboard focus to other areas of the comment in the accessibility pass (ie the whole comment box when viewing an existing comment), I think it might make more sense to handle focus at the comment level - if you see my ref logic in https://github.com/wagtail/wagtail/pull/6983, I use a ref for this - and instead of moving focus logic into the textarea, get it to forward a ref to the comment that can be used to focus. What do you think? At the very least we should remove the double focusing

Reply:
Oops, thanks for spotting that!

I copied this from Wagtail Localize for the auto-resize behaviour, just thought we might as well reuse the focusing logic too! happy to completely remove this if it makes things easier for #6983?